src/components/Table/rowRenderer.js

```diff
@@ -71,14 +75,30 @@ export default function defaultRowRenderer(
         <ExpandedComponent expandedData={data[openIndex]} />
       </div>
     );
-  } else if (index > openIndex) style.top = style.top + expandedHeight;
+  } else if (index > openIndex) {
+    style.top = style.top + expandedHeight;
+  }
+
+  // glyph hasn't been set, this isn't a selected row
+  if (glyph === undefined) glyph = 'fa-chevron-down';
+
+  // assume column can expand if expandedData is truthy
+  let expandVisualAid;
+  if (!!expandedData) {
+    expandVisualAid = (
+      <div style={{ position: 'absolute', right: '1rem' }}>
```

Comment:
`<span>` vs `<div>`?

Reply:
good question. I think `div`s are generally easier to position, so that would be what I lean towards.